pkgs/applications/networking/seaweedfs/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "seaweedfs";
+  version = "2.07";
+
+  src = fetchFromGitHub {
+    owner = "chrislusf";
+    repo = "seaweedfs";
+    rev = "${version}";
+    sha256 = "0yrfpj4ph9f321vqfn1zadv44pqa3ivjq9rx6gsz9hlv50gfaqn1";
+  };
+
+  vendorSha256 = "1ysqagi4y25bi84h5fhkdimnsigy43klf6hrcqn7q75382fb4bzn";
+
+  subPackages = [ "weed" ];
+
+  meta = with lib; {
```

Comment:
New derivations need maintainers. If you are going to proxy future updates to this derivation, you could add yourself as a maintainer :wink: .

Reply:
Getting somewhat mixed messages here (https://discourse.nixos.org/t/casual-nixpkgs-contributions/9607/2) - but happy to add myself as maintainer.